mqclient/queue.py

```diff
@@ -327,27 +331,28 @@ async def __aexit__(
         logging.debug(
             f"[MessageAsyncGeneratorContext.__aexit__()] exiting `with-as` block (exc:{exc_type})"
         )
-        if not (self.sub and self.gen):
+        if not (self._sub and self._gen):
```

Comment:
Here it looks like we're willing to throw a RuntimeError if either one of these is missing.

Reply:
see above